Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:3.8-slim
+WORKDIR /app
+COPY src/requirements.txt /app/
+COPY . /app
```

Comment:
Consider copying only the `requirements.txt` file first and running `pip install` before copying the rest of the application code. This can improve build caching efficiency, as the dependencies will only be reinstalled if the `requirements.txt` file changes.